### DISCUSSION TITLE
Solidity Metrics's Graphs Missing

### DISCUSSION THREAD

**Discussion by radin100** [Cyfrin/security-and-auditing-full-course-s23#28] [Q&A] [answered]
Why every time I run Solidity Metrix on src file there is no any Inheritance or Call Graph on the output document?

**Comment by PatrickAlphaC** [comment]
What are you seeing instead? Could you show a pic?

**Reply by radin100** [reply]
![Screenshot 2023-12-15 at 22 23 57](https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/123970228/8182f645-2642-490e-a055-27ed319c4142)

**Reply by radin100** [reply]
![Screenshot 2023-12-15 at 22 25 23](https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/123970228/dcdc9895-02ad-4d7d-9eaa-cf7d12bf8d96)

**Reply by PatrickAlphaC** [reply]
odd... Could be you're not selecting the correct files/folders... Or it's being very slow. Might be worth making an issue on their GitHub.

**Reply by radin100** [reply]
I see there is the right folder selected, because I see every other information about it, I guess I will sent them an issue. Thanks for help, Master!

**Comment by radin100** [comment, marked as answer]
nvm when I do it on smaller folders it works. Don't know why
